src/Analyzers/CSharp/CodeFixes/RemoveUnreachableCode/CSharpRemoveUnreachableCodeCodeFixProvider.cs

```diff
@@ -40,7 +40,7 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             // helpful, but shouldn't interfere with anything else the uesr is doing.
             var priority = IsSubsequentSection(diagnostic)
                 ? CodeActionPriority.Low
-                : CodeActionPriority.Medium;
+                : CodeActionPriority.Default;
```

Comment:
Lowest/Low/Default/High/Highest matches editor naming for this sort of thing (including the lightbulb priority class namign.  so i aligned on that for our public API).

Reply:
i also preferred 'Default' (versus 'Normal' or 'Medium') as it prevented the need to have a duplicate 'Default' member, or to explain what the 'Default' value for these enums was.  By just naming it that way, it all simplified (though it does impact the diff here).